targetplatform/EclipseSmartHome.setup

```diff
@@ -568,15 +568,15 @@
       <setupTask
           xsi:type="setup:PreferenceTask"
           key="/instance/org.eclipse.jdt.ui/sp_cleanup.make_private_fields_final"
-          value="false"/>
+          value="true"/>
       <setupTask
           xsi:type="setup:PreferenceTask"
           key="/instance/org.eclipse.jdt.ui/sp_cleanup.make_type_abstract_if_missing_method"
           value="false"/>
       <setupTask
           xsi:type="setup:PreferenceTask"
           key="/instance/org.eclipse.jdt.ui/sp_cleanup.make_variable_declarations_final"
```

Comment:
What does this one do?

Reply:
If you look at the screenshot in https://github.com/eclipse/smarthome/issues/3651#issue-235538193, this corresponds to the "outer" checkbox, i.e. it switches on the feature in general.

Reply:
Great!